### ISSUE TITLE
GEO Location Map Lovelace Card - problem on mobiles

### ISSUE THREAD

**Issue by DavidFW1960** [home-assistant/home-assistant-polymer#2430]
**Home Assistant release with the issue:**

0.85.0b0 and also b1
**Last working Home Assistant release (if known):**
new feature in 0.85.0

**Browser and Operating System:**

All browsers on iOS - the iOS beta app and chrome
**Description of problem:**

The card displays perfectly however on a mobile device tapping on an event does not show event details like it does on Win10 Chrome:
![image](https://user-images.githubusercontent.com/36390762/50935350-dbc5e780-14bf-11e9-9e8c-45ca660e4ff6.png)

Tapping on AR should show these details but does not on a mobile device.

![image](https://user-images.githubusercontent.com/36390762/50935368-eed8b780-14bf-11e9-9fdc-30d39ee9265e.png)

```
          - type: map
            geo_location_sources:
              - nsw_rural_fire_service_feed
            entities:
              - zone.home
```
**Javascript errors shown in the web inspector (if applicable):**
```

```

**Additional information:**

**Comment by DavidFW1960** [comment]
So seeing as you closed the other issue instead of this OLD one, it is not working on either iOS OR PC chrome now. No events showing at all on the event map.

**Comment by robbiet480** [comment]
@DavidFW1960 Just as a FYI for the future, replying to this issue stating that you were still experiencing this issue would've been the right move instead of opening a new issue. We like to keep the issue number low, and duplicating existing issues that haven't been marked fixed already doesn't help us achieve that goal 😺

**Comment by DavidFW1960** [comment]
this seems fixed in 0.89.0b3 so closing
